src/libs/actions/PersonalDetails/index.native.js

```diff
@@ -0,0 +1,329 @@
+import _ from 'underscore';
```

Comment:
I'm confused why this is necessary?

Reply:
This whole native file. It looks copied?

Reply:
@Julesssss The react-native-permissions library is not available in Web. So had to keep that logic separated out just for the native file.

Reply:
Couldn't we create an empty file for web, so that the remaining personal detail functions don't need to be duplicated?